<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1440</width>
    <height>900</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>18</pointsize>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow{background-color: rgb(255, 255, 255);}

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>1440</width>
     <height>900</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1440</width>
     <height>900</height>
    </size>
   </property>
   <widget class="QWidget" name="WidgetSerch" native="true">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>120</y>
      <width>1141</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
	background-color: rgba(128, 128, 128, 20);
}</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="widget_2" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>22</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{   
    border-top: 3px outset transparent;          /*边框全透明*/
	border-bottom: 3px outset transparent;
	border-right: 3px outset transparent;
	border-left: 3px outset transparent;  
    min-width: 0px;                            /* 控件最小宽度 */
    min-height: 0px;
    border-radius: 15px;               /*圆角*/
	background-color: rgba(0, 0, 0,0);
	color: rgb(57, 58, 58);
}</string>
          </property>
          <property name="text">
           <string>起点：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit">
          <property name="font">
           <font>
            <pointsize>22</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_3" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="leftMargin">
         <number>12</number>
        </property>
        <property name="topMargin">
         <number>9</number>
        </property>
        <property name="bottomMargin">
         <number>9</number>
        </property>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>22</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{   
    border-top: 3px outset transparent;          /*边框全透明*/
	border-bottom: 3px outset transparent;
	border-right: 3px outset transparent;
	border-left: 3px outset transparent;  
    min-width: 0px;                            /* 控件最小宽度 */
    min-height: 0px;
    border-radius: 15px;               /*圆角*/
	background-color: rgba(0, 0, 0,0);
	color: rgb(57, 58, 58);
}</string>
          </property>
          <property name="text">
           <string>终点：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_2">
          <property name="font">
           <font>
            <pointsize>22</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(3, 3, 3);</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_5" native="true">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>20</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <pointsize>22</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{   
    border-top: 3px outset transparent;          /*边框全透明*/
	border-bottom: 3px outset transparent;
	border-right: 3px outset transparent;
	border-left: 3px outset transparent;  
    min-width: 0px;                            /* 控件最小宽度 */
    min-height: 0px;
    border-radius: 15px;               /*圆角*/
	background-color: rgba(0, 0, 0,0);
	color: rgb(57, 58, 58);
}</string>
          </property>
          <property name="lineWidth">
           <number>0</number>
          </property>
          <property name="text">
           <string>出行方式：</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>10</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton">
          <property name="font">
           <font>
            <pointsize>22</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QRadioButton{   
    border-top: 3px outset transparent;          /*边框全透明*/
	border-bottom: 3px outset transparent;
	border-right: 3px outset transparent;
	border-left: 3px outset transparent;  
    min-width: 0px;                            /* 控件最小宽度 */
    min-height: 0px;
    border-radius: 15px;               /*圆角*/
	background-color: rgba(0, 0, 0,0);
	color: rgb(57, 58, 58);
}</string>
          </property>
          <property name="text">
           <string>火车</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>15</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_2">
          <property name="font">
           <font>
            <pointsize>22</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QRadioButton{   
    border-top: 3px outset transparent;          /*边框全透明*/
	border-bottom: 3px outset transparent;
	border-right: 3px outset transparent;
	border-left: 3px outset transparent;  
    min-width: 0px;                            /* 控件最小宽度 */
    min-height: 0px;
    border-radius: 15px;               /*圆角*/
	background-color: rgba(0, 0, 0,0);
	color: rgb(57, 58, 58);
}</string>
          </property>
          <property name="text">
           <string>飞机</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>60</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="font">
           <font>
            <pointsize>21</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{   
	background-color: rgb(149, 150, 150);
	color: rgb(57, 58, 58);
}

/* 鼠标悬停 */
QPushButton:hover{
	background-color: rgb(195, 197, 197);
}</string>
          </property>
          <property name="text">
           <string>搜索</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget_6" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>175</width>
      <height>800</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
	background-color: rgba(128, 128, 128, 20);
}</string>
    </property>
    <widget class="QLabel" name="appPic">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>50</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="appName">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>120</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>幼圆</family>
       <pointsize>18</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{   
    border-top: 3px outset transparent;          /*边框全透明*/
	border-bottom: 3px outset transparent;
	border-right: 3px outset transparent;
	border-left: 3px outset transparent;  
    min-width: 0px;                            /* 控件最小宽度 */
    min-height: 0px;
    border-radius: 15px;               /*圆角*/
	background-color: rgba(0, 0, 0,0);
	color: rgb(57, 58, 58);
}</string>
     </property>
     <property name="text">
      <string>交通查询系统</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>151</width>
       <height>60</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>18</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton{   
    border-top: 3px outset transparent;          /*边框全透明*/
	border-bottom: 3px outset transparent;
	border-right: 3px outset transparent;
	border-left: 3px outset transparent;  
    min-width: 0px;                            /* 控件最小宽度 */
    min-height: 0px;
    border-radius: 15px;               /*圆角*/
	background-color: rgba(0, 0, 0,0);
	color: rgb(57, 58, 58);
}

/* 鼠标悬停 */
QToolButton:hover{
	background-color: rgba(0, 0, 0,20);
}
</string>
     </property>
     <property name="text">
      <string> trip管理</string>
     </property>
     <property name="icon">
      <iconset resource="resource/res.qrc">
       <normaloff>:/images/trippic.png</normaloff>:/images/trippic.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>150</width>
       <height>60</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>18</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton{   
    border-top: 3px outset transparent;          /*边框全透明*/
	border-bottom: 3px outset transparent;
	border-right: 3px outset transparent;
	border-left: 3px outset transparent;  
    min-width: 0px;                            /* 控件最小宽度 */
    min-height: 0px;
    border-radius: 15px;               /*圆角*/
	background-color: rgba(0, 0, 0,0);
	color: rgb(57, 58, 58);
}

/* 鼠标悬停 */
QToolButton:hover{
	background-color: rgba(0, 0, 0,20);
}
</string>
     </property>
     <property name="text">
      <string> city管理</string>
     </property>
     <property name="icon">
      <iconset resource="resource/res.qrc">
       <normaloff>:/images/citypic.png</normaloff>:/images/citypic.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>42</width>
       <height>42</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>150</width>
       <height>60</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>18</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton{   
    border-top: 3px outset transparent;          /*边框全透明*/
	border-bottom: 3px outset transparent;
	border-right: 3px outset transparent;
	border-left: 3px outset transparent;  
    min-width: 0px;                            /* 控件最小宽度 */
    min-height: 0px;
    border-radius: 15px;               /*圆角*/
	background-color: rgba(0, 0, 0,0);
	color: rgb(57, 58, 58);
}

/* 鼠标悬停 */
QToolButton:hover{
	background-color: rgba(0, 0, 0,20);
}
</string>
     </property>
     <property name="text">
      <string> 账户管理</string>
     </property>
     <property name="icon">
      <iconset resource="resource/res.qrc">
       <normaloff>:/images/userpic.png</normaloff>:/images/userpic.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
    </widget>
    <widget class="QWidget" name="WidgetBtn" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>650</y>
       <width>151</width>
       <height>114</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{   
    border-top: 3px outset transparent;          /*边框全透明*/
	border-bottom: 3px outset transparent;
	border-right: 3px outset transparent;
	border-left: 3px outset transparent;  
    min-width: 0px;                            /* 控件最小宽度 */
    min-height: 0px;
    border-radius: 15px;               /*圆角*/
	background-color: rgba(0, 0, 0,0);
	color: rgb(57, 58, 58);
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="SignInBtn">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{   
    border-radius: 15px;               /*圆角*/
    background-color: rgba(128, 128, 128, 20);
    color:rgb(57, 58, 58);
}

/* 鼠标悬停 */
QPushButton:hover{
	color: rgb(205, 207, 207);
}
</string>
        </property>
        <property name="text">
         <string>管理员登陆</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="ExitBtn">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{   
    border-radius: 15px;               /*圆角*/
    background-color: rgba(128, 128, 128, 20);
    color:rgb(57, 58, 58);
}

/* 鼠标悬停 */
QPushButton:hover{
	color: rgb(205, 207, 207);
}</string>
        </property>
        <property name="text">
         <string>  退   出  </string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resource/res.qrc"/>
 </resources>
 <connections/>
</ui>
